db/data/20230217112836_add_synonyms_data_to_provider.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AddSynonymsDataToProvider < ActiveRecord::Migration[7.0]
+  def up
+    csv_filepath = Rails.root.join(__FILE__.gsub('.rb', '.csv'))
```

Comment:
I think this should be a rake task and then we have a migration to call it? Are we handling the import rules? Eg stop words etc? Re: the rake task, we may need to re-import this list if we ever want to add additional synonyms

Reply:
Done [here](https://github.com/DFE-Digital/publish-teacher-training/pull/3363/commits/c5ca66cf368e8e0c25543862285ac5404abadb92). We can run this in prod when we merge.